README.md

```diff
@@ -24,40 +24,51 @@ cd sample-go-rabbitmq
 
 #### Install RabbitMQ via Helm
 
-Since the Helm stable repositoty was migrated to the Bitnami repository (https://github.com/helm/charts/tree/master/stable/rabbitmq), add the Bitnami repo and use it during the installation (bitnami/<chart> instead of stable/<chart>) 
+Since the Helm stable repositoty was migrated to the [Bitnami Repository](https://github.com/helm/charts/tree/master/stable/rabbitmq), add the Bitnami repo and use it during the installation (bitnami/\<chart\> instead of stable/\<chart\>)
```

Comment:
I'm not 100% sure about escaping `<chart>`. I know that linter sows an error but I think github will not understand it if it's escaped (you can introduce html tags in md). Please remove the escape bars please 🙏